src/extension.ts

```diff
@@ -133,6 +133,11 @@ export function activate(context: ExtensionContext) {
     context.subscriptions.push(commands.registerCommand(GaugeVSCodeCommands.GenerateStub, (code: string) => {
         return generateStub(clients, code);
     }));
+
+    context.subscriptions.push(commands.registerCommand(GaugeVSCodeCommands.ExtractConcept, (rangeInfo: Object) => {
```

Comment:
Instead of this, can we embrace a pattern where `context` is passed to a `Command`, in this case, say, `ExtractCommand` and registration can be part of the command?

This can also avoid exporting of `extractConcept`.

See `WelcomeProvider` for reference.